src/kolmogorov_solve.jl

```diff
@@ -0,0 +1,53 @@
+struct NNKolmogorov{C,O,S} <: NeuralNetDiffEqAlgorithm
+    chain::C
+    opt::O
+    sdealg::S
+   
+end
+NNKolmogorov(chain  ; opt=Flux.ADAM(0.1) , sdealg = EM()) = NNKolmogorov(chain , opt , sdealg)
+ 
+function DiffEqBase.solve(
+    prob::KolmogorovPDEProblem,
+    alg::NeuralNetDiffEqAlgorithm;
+    abstol = 1f-6,
+    verbose = false,
+    maxiters = 300,
+    save_everystep = false,
+    dt,
+    kwargs...
+    )
+
+    tspan = prob.tspan
+    phi = prob.phi
+    xspan = prob.xspan
+    sigma = prob.sigma
+    μ = prob.μ
+    d = prob.d
+    ts = tspan[1]:dt:tspan[2]
+    N = size(ts)
+    T = tspan[2]
+    #hidden layer
+    chain  = alg.chain
+    opt    = alg.opt
+    sdealg = alg.sdealg
+    ps     = Flux.params(chain)
+    xi     = rand(Uniform(xspan[1] , xspan[2]), d , N[1] )
+    #Finding Solution to the SDE having initial condition xi. Y = Phi(S(X , T))
+    sdeproblem = SDEProblem(μ,sigma,xi,tspan)
+    sol = solve(sdeproblem, sdealg ,dt=dt , save_everystep=false , kwargs...)
+    x_sde = sol[end]
+    y = phi(x_sde)
+    data   = Iterators.repeated((xi , y), maxiters)
```

Comment:
you're only testing on one data point: there's no way that is going to be the algorithm. It's likely looping over the solve?

Reply:
xi is of shape [dimensions , number of data points] , and it is passed as an initial condition for sde. so same dimensions for x_sde. y is of the shape [dim_y , number of data points] .
Instead of looping over again for sde , I added all data points in initial condition.

Reply:
![Screenshot 2020-02-27 at 12 36 55 PM](https://user-images.githubusercontent.com/43771652/75420343-e2cae600-595d-11ea-9159-ada2d0248f16.png)
 Although this can be done too.

Reply:
but you're matching the network to only one run, instead of always resampling. I guess if the number of samples is high enough that's okay.

We'll want to move this to https://docs.juliadiffeq.org/latest/features/ensemble/ to get to higher dimensions and parallelism.

Reply:
but you're matching the network to only one run, instead of always resampling. I guess if the number of samples is high enough that's okay.

We'll want to move this to https://docs.juliadiffeq.org/latest/features/ensemble/ to get to higher dimensions and parallelism.